AppKit/CPView.j

```diff
@@ -239,6 +240,11 @@ var CPViewHighDPIDrawingEnabled = YES;
     BOOL                _allowsVibrancy         @accessors(property=allowsVibrancy);
     CPAppearance        _appearance             @accessors(getter=appearance);
     CPAppearance        _effectiveAppearance;
+
+    CPMutableArray      _trackingAreas          @accessors(getter=trackingAreas, copy);
+    BOOL                _inhibitUpdateTrackingAreasNotifications;
```

Comment:
Please rename this to _inhibitUpdateTrackingAreas, there are not CPNotifications being sent.